### PR TITLE
Payload invites

### PR DESCRIPTION
**Pull request by josmas**
email field removed, and upgrade to postgrator (for cross platform checksums) cc/ @tangentfairy

### CODE REVIEW

**Comment by tangentfairy** [comment]
:+1: Looks good!

File: dojos.js [hunk 1]
```diff
@@ -554,12 +554,25 @@ module.exports = function (options) {
     }
   }
 
+  function purgeInviteEmails (invitesArray) {
+    return _.map(invitesArray, function (invite) { delete invite.email; return invite; });
```
Comment (ckiss) [inline dojos.js]: @josmas I would not use `delete` but _.omit instead
cc @tangentfairy

Comment (josmas) [inline dojos.js]: hey @ckiss Can you share the reason with us?

Comment (tangentfairy) [inline dojos.js]: I think lo-dash is supposed to be faster, but maybe there is another reason!

Comment (josmas) [inline dojos.js]: Doesn't seem to be the case in [browsers](https://jsperf.com/delete-v-omit/2), but this is node code.

**Comment by ckiss** [comment]
@josmas @tangentfairy I think it's faster and safer(delete will return an error in strict mode if can't delete that property for any reason). Also if you'll want to delete more of those properties in the future will be easier to do it using _.omit and you'll get easier to read code.

**Comment by ckiss** [comment]
Also one more thing I think should be taken in consideration: that command is called `cmd_list`, but besides the list it does now other things (in this case just removes those emails which is not such a big thing, but in other cases is more than that) which is not the best thing to do, cause some other contributor might need the dojos with those emails and all in the backend and he'll call cmd_list but surprise... there are no emails in the results. What will he do in this case? 

What I'm saying is that commands like: list, load, save delete should be kept as clean as possible (they will do only those fixed operations) and just create new commands with more adequate names.

**Comment by tangentfairy** [comment]
That makes sense @ckiss  - we had to get something out to fix the security issue but it would be good to refactor this to be a separate endpoint.

**Comment by josmas** [comment]
@ckiss I agree on the multiple responsibilities of the endpoint, there are a number of commands in that list that do specific things, which is good, although the file is getting really big (would be good to separate into smaller files too).

In this case though, I don't think that the email field should be **ever** returned in an API call, as it should only be used to send the invitation, so it might be that it does not need to be stored in the DB in first place?

**Comment by ckiss** [comment]
@josmas splitting the file would be awesome (events service might be a good example of how to do that)

I agree with the fact the the emails shouldn't be returned by any API call, but that command should return everything if called from inside the service itself. TBH I don't know in this specific case if the emails should be stored in the DB or not. Might worth hearing Ursula's opinion on this one. Maybe they are needed for statistics or stuff like that, I have no clue

**Comment by tangentfairy** [comment]
I think the user_id of the mentor is enough, we don't really need to have the email address in there. If we want to do statistics on it later then we still know which user requested the invite.

**Pull request merged**